src/sentry/api/serializers/models/sentry_app.py

```diff
@@ -30,22 +30,13 @@ def serialize(self, obj, attrs, user):
         if is_active_superuser(env.request) or (
             hasattr(user, 'get_orgs') and obj.owner in user.get_orgs()
         ):
-            if obj.is_internal:
-                install = obj.installations.first()
-                data.update({
-                    'installation': {
-                        'uuid': install.uuid,
-                    },
-                    'token': install.api_token.token,
-                })
-            else:
-                data.update({
-                    'clientId': obj.application.client_id,
-                    'clientSecret': obj.application.client_secret,
-                    'owner': {
-                        'id': obj.owner.id,
-                        'slug': obj.owner.slug,
-                    },
-                })
+            data.update({
+                'clientId': obj.application.client_id,
+                'clientSecret': obj.application.client_secret,
```

Comment:
@MeredithAnya I am a bit confused by this change. Why would we send back the `clientId` and the `clientSecret` for internal integrations?

Reply:
I'm going to change this to be `if not obj.internal` then return the id and secret